modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -427,6 +427,32 @@ The Mule upgrade tool requires both the current Mule instance and the new Mule i
 
 Perform a manual upgrade to ensure that the current and new Mule instances are in version 4.1.5 or later before attempting to upgrade using the Mule upgrade tool.
 
+[[upgrade_clustered_mule_instances]]
+== Upgrading Clustered Mule Instances
+
+The Mule upgrade tool does not automatically support upgrading clustered Mule instances. To upgrade clustered Mule instances, you have to manually upgrade each node using the tool.
+
+=== Before You Begin
+
+Before starting your upgrade:
+
+* Copy the new Mule runtime (with the version to be upgraded to) and Mule upgrade tool instances to each node that must be upgraded, to ensure less downtime.
+* Follow the procedures described in the above <<before_you_begin>> section.
+
+=== Upgrade or Update Mule
+
+To upgrade or update your clustered Mule instances:
+Follow the procedures as in the stand-alone instance upgrade described in the above <<upgrade_or_update_mule>> section. Start from the server node and continue with the other nodes until each node is upgraded to the same version.
```

Comment:
Regarding all the sections where we are pointing back to the same procedure above (saying, "Follow the procedures described in the above <<roll_back_an_upgrade-or-update>> section"), I chatted with Hanna who suggested using includes instead of links here. The idea was that here we are talking to people who are upgrading clusters. Above they are upgrading a single instance. Includes will avoid the need to click the links for the steps.

Reply:
100% we should avoid pointing people to different sections, even if the content is identical (it's hard for the user to keep track and doesn't provide a targeted experience). Definitely use includes and duplicate the relevant content in both places.

Reply:
I don't know about that. That's not my way of thinking, and programming, so I will have to trust in you about this, but is it not an easy thing for me to do it this way 🤣

Reply:
Creating includes.